### ISSUE TITLE
[tooltip] enable accessibility

### ISSUE THREAD

**Issue by tlouisse** [ing-bank/lion#754]
OverlayController provides `isTooltip` and `invokerRelation` to configure the right a11y attributes (role, ariadescribedby, arialabelledby) for `LionTooltip`.
For some reason, these options are not enabled inside LionTooltip.

See https://polymer.slack.com/archives/CJGFWJN9J/p1589446170190000

https://github.com/ing-bank/lion/pull/726 contains a fix, it also properly cleans up a11y attrs on teardown.
